numpy/_typing/_scalars.py

```diff
@@ -1,3 +1,4 @@
+# pyright: strict
```

Comment:
Is this safe from a user experience?

Reply:
oh, my mistake, slipped in while testing.

Reply:
absolutely right, just removed it.